typings/optimist/optimist.d.ts

```diff
@@ -7,49 +7,49 @@
 
 declare module "optimist" {
 
-    function optimist(args: string[]): optimist.Optimist;
+	function optimist(args: string[]): optimist.Optimist;
 
-    module optimist {
-        export interface Optimist {
-            default(name: string, value: any): Optimist;
-            default(args: Object): Optimist;
+	module optimist {
+		export interface Optimist {
+			default(name: string, value: any): Optimist;
+			default(args: Object): Optimist;
 
-            boolean(name: string): Optimist;
-            boolean(names: string[]): Optimist;
+			boolean(name: string): Optimist;
+			boolean(names: string[]): Optimist;
 
-            string(name: string): Optimist;
-            string(names: string[]): Optimist;
+			string(name: string): Optimist;
+			string(names: string[]): Optimist;
 
-            wrap(columns: number): Optimist;
+			wrap(columns: number): Optimist;
 
-            help(): void;
-            showHelp(fn?: Function): void;
+			help(): void;
+			showHelp(fn?: Function): void;
 
-            usage(message: string): Optimist;
+			usage(message: string): Optimist;
 
-            demand(key: string): Optimist;
-            demand(key: number): Optimist;
-            demand(key: string[]): Optimist;
+			demand(key: string): Optimist;
+			demand(key: number): Optimist;
+			demand(key: string[]): Optimist;
 
-            alias(key: string, alias: string): Optimist;
+			alias(key: string, alias: string): Optimist;
 
-            describe(key: string, desc: string): Optimist;
+			describe(key: string, desc: string): Optimist;
 
-            options(key: string, opt: Object): Optimist;
+			options(key: string, opt: Object): Optimist;
 
-            check(fn: Function): void;
+			check(fn: Function): void;
 
-            parse(args: string[]): Optimist;
+			parse(args: string[]): Optimist;
 
-            argv: Argv;
-        }
+			argv: Argv;
+		}
 
-        export interface Argv extends Object {
-            _: string[];
-        }
+		export interface Argv extends Object {
+			_: string[];
+		}
 
-        export var argv: any;
-    }
+		export var argv: any;
```

Comment:
I couldn't find out any better way to typed the `argv` parameter. Welcome any suggestion.